CONTRIBUTING.md

```diff
@@ -1,13 +1,608 @@
 
-Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.  Please see https://git.k8s.io/community/CLA.md for more info
+_As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities._
 
-### Contributing A Patch
+- [Getting Started](#getting-started)
+- [Mentorship](#mentorship)
+- [Contact Information](#contact-information)
+- [Contributing Code](#contributing-code)
+- [Coding Style](#coding-style)
+  - [Explain 'nolint' Directives](#explain-nolint-directives)
+  - [Use Descriptive Variable Names Sparingly](#use-descriptive-variable-names-sparingly)
+  - [Don't Wrap Function Signatures](#dont-wrap-function-signatures)
+  - [Return Early](#return-early)
+  - [Wrap Errors](#wrap-errors)
+  - [Scope Errors](#scope-errors)
+  - [Prefer Table Driven Tests](#prefer-table-driven-tests)
+- [Establishing a Development Environment](#establishing-a-development-environment)
+- [Attribution](#attribution)
 
-1. Submit an issue describing your proposed change to the repo in question.
-1. The [repo owners](OWNERS) will respond to your issue promptly.
-1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
-1. Fork the desired repo, develop and test your code changes.
-1. Submit a pull request.
+## Getting Started
+
+We have full documentation on how to get started contributing here:
+
+<!---
+If your repo has certain guidelines for contribution, put them here ahead of the general k8s resources
+-->
+
+- [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
+- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
+- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet) - Common resources for existing developers
+
+## Mentorship
+
+- [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
+
+## Contact Information
+
+<!-- TODO(contact): Dedupe with README and k/community -->
+
+- [Slack channel](https://kubernetes.slack.com/messages/sig-release)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+
+## Contributing Code
+
+To contribute bug fixes or features to Crossplane:
```

Comment:
Thanks Arnaud!
Adjusted your suggestion to "To contribute bug fixes or features to SIG Release repositories:", since this will be a general guide for all SIG Release repos to reference in their CONTRIBUTING.md.